dev-support/docker/docker/Dockerfile

```diff
@@ -53,23 +53,30 @@ RUN sed -ri 's/UsePAM yes/UsePAM no/g' /etc/ssh/sshd_config
 # Install python, nodejs and npm
 RUN yum -y install http://dl.fedoraproject.org/pub/epel/6/x86_64/epel-release-6-8.noarch.rpm
 RUN yum -y install nodejs npm --enablerepo=epel
-RUN npm install -g brunch@1.7.13
+RUN npm install -g brunch@1.7.17
+
+RUN touch /etc/yum.repos.d/ambari.repo
```

Comment:
Is this is included because server boot process expects the repository file, the issue has been fixed in https://issues.apache.org/jira/browse/AMBARI-12682